includes/controls/settings-footer.php

```diff
@@ -0,0 +1,536 @@
+<?php /**
+ * Hello Elementor additional Footer options
+ *
+ * @package HelloElementor
+ */
+
+namespace Hello_Elementor;
+
+use Elementor\Plugin;
+use Elementor\Controls_Manager;
+use Elementor\Group_Control_Background;
+use Elementor\Group_Control_Typography;
+use Elementor\Core\Responsive\Responsive;
+use Elementor\Core\Kits\Documents\Tabs\Tab_Base;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+class Hello_Settings_Footer extends Tab_Base {
```

Comment:
Change class name and usage to `Settings_Footer` - the file name should be identical to the class,
Do this also for `Settings_Header`.

Reply:
Changed.